http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -387,6 +378,17 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       _ <- logger.info(ctx.loggingCtx)(
         s"Finished app creation for app ${app.appName.value} in cluster ${gkeClusterId.toString}"
       )
+      pvcs <- kubeService.getPersistentVolumeClaims(gkeClusterId, KubernetesNamespace(app.appResources.namespace.name))
+      galaxyPvc = pvcs.find(pvc => pvc.getMetadata.getName == s"${app.release.asString}-galaxy-pvc")
+      cvmfsPvc = pvcs.find(pvc => pvc.getMetadata.getName == s"${app.release.asString}-cvmfs-alien-cache-pvc")
+      _ <- (galaxyPvc, cvmfsPvc).tupled
+        .fold(F.raiseError[Unit](new LeoException("Fail to retrieve pvc ids", traceId = Some(ctx.traceId)))) {
+          case (gp, cp) =>
+            appQuery
+              .updatePvcIds(params.appId, PvcId(gp.getMetadata.getUid), PvcId(cp.getMetadata.getUid))
+              .transaction
+              .void
+        }
```

Comment:
Not sure if this logic will work. I think we need to distinguish between two possible cases and take different actions accordingly, i.e., 

1. When creating a Galaxy app **and** a **new** disk, do as above.
2. When creating a Galaxy app **reattaching** an **existing** disk, get the PVC IDs from `PERSISTENT_DISK` table (since we must have executed 1. at some point when first creating the disk)

[link to the related section of the design doc](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/1184464905/2021-01-05+Galaxy+Persistence#Explanation)

Reply:
we don't have any code for re-attaching disk yet(it'll be in the next PR)...this is PR's goal is really just to retrieve the disk IDs and persist them

Reply:
actually, I might just update this to reflect re-attaching in the same PR...

Reply:
I end up making this particular change since it doesn't inflate this PR that much...but don't worry too much about re-attaching logic yet in this PR since it'll be addressed in a follow up PR...